pages/index.tsx

```diff
@@ -1,16 +1,9 @@
-import * as React from 'react';
+import React from 'react';
 import SomeComponent from 'client/components/SomeComponent';
-import styled from 'styled-components';
-
-const Title = styled.h1`
-  font-size: 50px;
-  color: ${({ theme }) => theme.colors.primary};
-`;
 
```

Comment:
can we add back in some example of using styled components for future contributors?

Reply:
@timmyichen Yup, done.